src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnCache.java

```diff
@@ -111,11 +111,11 @@
     this.topLevelOutputs = Preconditions.checkNotNull(topLevelOutputs, "topLevelOutputs");
   }
 
-
   @Override
   public CacheHandle lookup(Spawn spawn, SpawnExecutionContext context)
       throws InterruptedException, IOException, ExecException {
-    if (!Spawns.mayBeCached(spawn) || !Spawns.mayBeExecutedRemotely(spawn)) {
+    if (!Spawns.mayBeCached(spawn)
```

Comment:
```suggestion
    if (!Spawns.mayBeCachedRemotely(spawn))
```
should be enough here. 
`!Spawns.mayBeExecutedRemotely(spawn)` - this one we do not need because this is the purpose of the PR - allow remote caching of the local actions.

What was the reason to check `isDiskCache(options)` here as well?

Reply:
My reasoning was that the disk cache is technically a local cache, so it wouldn't be disabled by the `no-remote-cache` requirement. (see my comment above: https://github.com/bazelbuild/bazel/pull/8710#discussion_r301185517)

Reply:
This thread is resolved (see https://github.com/bazelbuild/bazel/pull/8710#discussion_r302165910)